completions/ssh.completion.sh

```diff
@@ -18,6 +20,17 @@ function _omb_completion_ssh {
     COMPREPLY=($(compgen -W "$(grep ^Host "$HOME/.ssh/config" | awk '{for (i=2; i<=NF; i++) print $i}' )" "${options[@]}"))
   fi
 
+  # check if .ssh/config contains Include options
+  for additional_include_option in $(awk -F' ' '/^Include/{print $2}' "$HOME/.ssh/config" 2>/dev/null) ;do
+    # relative or absolute path, if relative transforms to absolute
+    [[ "${additional_include_option:0:1}" == "/" ]] ||additional_include_option="$HOME/.ssh/$additional_include_option"
```

Comment:
One can use glob-pattern matching in `[[ ... ]]`.

```suggestion
    [[ "$additional_include_option" == /* ]] ||additional_include_option="$HOME/.ssh/$additional_include_option"
```

Reply:
c9198a7